src/AdminConsole/Program.cs

```diff
@@ -113,6 +113,7 @@ void RunTheApp()
     services.ConfigureApplicationCookie(o =>
     {
         o.Cookie.Name = "AdminConsoleSignIn";
+        o.Cookie.SameSite = SameSiteMode.Strict;
```

Comment:
I think this is good to harden the security. We already do CSRF protection using the antiforgery-token, but having multiple levels of protection is good.

Have you tested the behaviour of the application when clicking links to the admin console? There are many places where we link or could be linking to our app (Email with magic link, from documentation, notification emails).

Are those links honoured? Or will the user be sent to a unauthorised-page, and then redirect to the index page?

![CleanShot 2024-02-02 at 13 01 36](https://github.com/bitwarden/passwordless-server/assets/357283/15522ec3-c06e-4321-a559-99b6b2867f74)

Reply:
@jrmccannon or @Tyrrrz Do either of you guys want to help me test this on Monday with magic links? I can test the other scenarios.

Reply:
Running locally, magic links worked fine for logging into AdminConsole.

Reply:
@jrmccannon You tested a magic link from a file open in vscode/ide, not from another website, like gmail.com -- right?